src/routes/components/Block.tsx

```diff
@@ -90,9 +90,9 @@ export default function BlockPage() {
                   backgroundColor: "rgba(0, 150, 255, 0.2)",
                   whiteSpace: "nowrap",
                   width: "5rem",
-                }}
-              >
-                Item 2
+                }}>
+                <p>Item 2</p>
```

Comment:
Instead of `p` tags can you use a `br` instead. This will prevent the padding on the bottom item from moving this off center.